scripts/fuzz_shell.js

```diff
@@ -178,9 +178,10 @@ function callFunc(func) {
     /* await */ func();
     return 0;
   } catch (e) {
-    // The exception must exist, and not behave oddly when we access a
-    // property on it. (VM bugs could cause errors here.)
-    e.a;
+    // The exception might be a JS null, but otherwise it must be valid to check
+    // if a property exists on it (VM bugs could cause errors here, specifically
+    // if a wasm exception is caught here, and it is not represented properly).
+    if (e !== null) e.a;
```

Comment:
Randomly saw this PR in my feed, just wanted to say that 1) this is not enough, you can also `throw undefined` and this check won't catch it and 2) you can just use `e?.a` instead, which will catch both `null` and `undefined` in a nice short syntax.

Reply:
Hm, unless it's not possible to express `throw undefined` from Wasm?

Reply:
Yeah, all wasm can do is throw null (as `ref.null noextern`; the only way to get `undefined` would be to import it from JS, which we don't have in our JS).

I am worried about `e?.a` because of how it might interact with a wasm exception here. I guess there could be VM bugs both ways... but `e.a` was what was suggested to me.